src/quo2/components/dividers/divider_label.cljs

```diff
@@ -14,58 +14,69 @@
    counter-value -> number
    increase-padding-top? -> boolean
    blur? -> boolean"
-  [{:keys [label chevron-position counter-value increase-padding-top? blur? container-style]}]
+  [{:keys [label
```

Comment:
Added `padding-bottom` and `on-press` and `chevron-icon` to the component

Reply:
The docstring is probably outdated since you added new options.